cszczepaniak/cribbly-backend
# Add update game endpoint

# What I added (link any issues addressed)
Closes #30 

# How to test it

- Make sure a game exists
- Call ```PUT https://localhost:5001/api/game/{id}```

Body:

```
{
    "id": int,
    "teams": [
        {
            "id": int,
            "name": "string",
            "division": null,
            "players": null,
            "gameScores": null,
            "playInGames": null,
            "bracketGames": null,
            "wins": int,
            "losses": int,
            "totalScore": int,
            "ranking": int,
            "seed": int,
            "inTournament": bool
        },
        {
            "id": int,
            "name": "string",
            "division": null,
            "players": null,
            "gameScores": null,
            "playInGames": null,
            "bracketGames": null,
            "wins": int,
            "losses": int,
            "totalScore": int,
            "ranking": int,
            "seed": int,
            "inTournament": bool
        }
    ],
    "winner": {
        "id": int,
        "name": "string",
        "division": null,
        "players": null,
        "gameScores": null,
        "playInGames": null,
        "bracketGames": null,
        "wins": int,
        "losses": int,
        "totalScore": int,
        "ranking": int,
        "seed": int,
        "inTournament": bool
    },
    "scoreDifference": int,
    "gameRound": int
}

```

- See that the updated game gets returned
